go.work

```diff
@@ -12,3 +12,4 @@ use (
 )
 
 //replace github.com/wundergraph/graphql-go-tools/v2 => ../graphql-go-tools/v2
+//replace github.com/wundergraph/astjson => ../astjson
```

Comment:
Did you intend to leave this in?

Reply:
yes, I think it's useful in case someone wants to work on astjson, similarly to go tools